lib/requestsession.py

```diff
@@ -104,12 +104,15 @@ def do_request(self, method, url, data=None):
         Helper class to regroup requests and handle exceptions at the same
         location
         """
+        headers = {
+            'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.146 Safari/537.36'
+        }
```

Comment:
Would it be possible to avoid hardcoded User Agent and rather use a flag on the command line? (e.g. --ua or --user-agent flag). It would need to add the argument in the ArgumentParser, and add a setter or a parameter to RequestSession.__init__. Shouldn't be too hard. Maybe it would be also useful to add a little set of common User Agents in the code and let the user choose in it.

Reply:
i'm working on the changes, i have python as an adventure since i always programmed PHP haha

Reply:
Take your time.
I haven't updated it for 9 months anyway :D

Reply:
@MickaelWalter I hard-coded a fix onto my copy a while back, but I agree that a multi-faceted approach (with the three flags you mentioned) works best. Also wanted to say this tool has been incredibly useful in my work. Thanks for building it.